src/mongo/db/fts/fts_tokenizer.h

```diff
@@ -75,6 +75,12 @@ class FTSTokenizer {
      */
     static const Options kGenerateDiacriticSensitiveTokens = 1 << 2;
 
+    /**
+     * Generate delimiter based tokens for N-Gram
+     * This is need for phrase search(Everything is compared as phrase search in N-gram)
+     */
+    static const Options kGenerateDelimiterTokensForNGram = 1 << 3;
```

Comment:
If MongoDB will ever add additional options here they will use the same value (1 << 3) and you will be forced to change the value of kGenerateDelimiterTokensForNGram. So I think to avoid that it would be smart to change (1 << 3) here to some bigger value. Since Options is uint8_t now I would use (1 << 7)